README.rst

```diff
@@ -1,7 +1,26 @@
+flit
```

Comment:
> The README is included as the front page of the docs, so please leave off this title. I prefer READMEs to open with text anyway.

Althrough I don't necessarily agree on this sentiment, I'm not gonna argue about this. Fixed.

> We'll probably need to remove the logo from the docs so that it's not there twice, as well.

I'm not sure what you're saying, you should definitely leave a vector-graphics version of your logo around in any case.